packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -493,261 +489,303 @@ export class StatsAnalyzer extends EventsScope {
-              );
-            }
           }
 
-          this.emitStartStopEvents(
-            mediaType,
-            previousStats.framesDecoded,
-            currentStats.framesDecoded,
-            false
-          );
-        }
-
-        if (this.meetingMediaStatus.expected.sendShare && mediaType.includes('video-share-send')) {
-          // compare share stats sent
-
-          currentStats = this.statsResults[mediaType].send;
-          previousStats = this.lastStatsResults[mediaType].send;
-
           if (
-            currentStats.totalPacketsSent === previousStats.totalPacketsSent ||
-            currentStats.totalPacketsSent === 0
+            this.statsResults.resolutions['video-share-send'].send.framesSent ===
+              this.lastStatsResults.resolutions['video-share-send'].send.framesSent ||
+            this.statsResults.resolutions['video-share-send'].send.framesSent === 0
           ) {
             LoggerProxy.logger.info(
-              `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} RTP packets sent`
+              `StatsAnalyzer:index#compareLastStatsResult --> No share frames sent`
             );
-          } else {
-            if (
-              currentStats.framesEncoded === previousStats.framesEncoded ||
-              currentStats.framesEncoded === 0
-            ) {
-              LoggerProxy.logger.info(
-                `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames getting encoded`
-              );
-            }
-
-            if (
-              this.statsResults.resolutions[mediaType].send.framesSent ===
-                this.lastStatsResults.resolutions[mediaType].send.framesSent ||
-              this.statsResults.resolutions[mediaType].send.framesSent === 0
-            ) {
-              LoggerProxy.logger.info(
-                `StatsAnalyzer:index#compareLastStatsResult --> No ${mediaType} frames sent`
-              );
-            }
           }
         }
+      }
 
-        if (this.meetingMediaStatus.expected.sendShare && mediaType.includes('video-share-recv')) {
-          // TODO:need to check receive share value
-          // compare share stats received
-          currentStats = this.statsResults[mediaType].recv;
-          previousStats = this.lastStatsResults[mediaType].recv;
+      if (this.meetingMediaStatus.expected.sendShare) {
+        // TODO:need to check receive share value
+        // compare share stats received
+        const currentPacketsReceived = getCurrentStatsTotals(
+          'video-share-recv',
+          'totalPacketsReceived'
+        );
+        const currentPacketsSent = getCurrentStatsTotals('video-share-recv', 'totalPacketsSent');
+        const previousPacketsReceived = getPreviousStatsTotals(
+          'video-share-recv',
+          'totalPacketsReceived'
+        );
+        const currentFramesReceived = getCurrentResolutionsStatsTotals(
+          'video-share-recv',
+          'totalFramesReceived'
```

Comment:
question: I don't see 'totalFramesReceived' referenced anywhere else in the code, so how/where is it being set?

Reply:
Updated to `framesReceived` along with a couple other instances of this.